specs/simple-serialize.md

```diff
@@ -70,7 +73,13 @@ assert value in (True, False)
 return b"\x01" if value is True else b"\x00"
 ```
 
-### Vectors, containers, lists
+### `"null"`
+
+```python
+return b""
```

Comment:
I think this would run into the same problem as [this one](https://github.com/ethereum/eth2.0-specs/issues/854). So either we should define null as some non-empty string (e.g., `b"\x00"`) or allow it only as a container or vector element.

Reply:
Yes, I agree. I think null only makes sense inside an option type and then it should just be an empty string (so it does not take any space in serialization). And we should forbid all other empty container objects.

Reply:
To clarify, you mean keep this code exactly as is, so in any actual implementation the rules for union types below would apply, and a `null` would resolve as just the 4 bytes representing whatever its index in the union type is?

Reply:
Yes, exactly.